src/python/pants/backend/debian/target_types.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.goals.package import OutputPathField
+from pants.engine.target import (
+    COMMON_TARGET_FIELDS,
+    DictStringToStringField,
+    SpecialCasedDependencies,
+    StringField,
+    Target,
+)
+from pants.util.docutil import doc_url
+
+
+class DebianControlFile(StringField):
+    alias = "control"
+    help = "Path to a Debian control file for the package to be produced."
+
+
+class DebianSymlinks(DictStringToStringField):
+    alias = "symlinks"
+    help = "Symlinks to create for each target being packaged."
+
+
+class DebianInstallPrefix(StringField):
+    alias = "install_prefix"
+    default = "/opt"
+    help = "Absolute path to a directory where Debian package will be installed to."
+
+
+class DebianPackageDependencies(SpecialCasedDependencies):
+    alias = "packages"
```

Comment:
Thanks! I know Benjy mentioned devil's advocate that he thinks this should maybe be a normal `Dependencies` field, but I recommend we start with this. 

That way we don't need to worry yet about code that differentiates between these "special" dependencies that get built via `./pants package` vs. "normal" dependencies where we just copy in the loose source files.

Reply:
I see. Then I leave this part of code as it is now, right?

Reply:
Yes, i think it will be easier to work with.